topi/python/topi/arm_cpu/conv2d.py

```diff
@@ -39,10 +39,11 @@ def decl_spatial_pack(cfg, data, kernel, strides, padding, layout, out_dtype):
 def schedule_conv2d_nchw_arm_cpu(cfg, outs):
     """TOPI schedule callback"""
     s = tvm.create_schedule([x.op for x in outs])
+    scheduled_ops = []
```

Comment:
You should handle this logic in the utility function `traverse_inline`.
Currently the traverse inline logic takes much redundant code in TOPI. I recommend that developers should switch to use this utility function.

Reply:
Yeah it was pain to add all those scheduled_ops stuff to all backends.

But it is not a bug, right?

Reply:
Here is my local version. Possibly you can update it for arm_cpu only in your NNVM fusion PR.

```python
def traverse_inline(s, final_op, callback):
    """Traverse computation graph and do auto inline

    Parameters
    ----------
    s: schedule
        The schedule
    final_op: Operation
        The final output operator.
    callback: callable
        The callback function on each op
    """
    visited = set()

    def _traverse(op):
        if op in visited:
            return
        visited.add(op)
        if tag.is_injective(op.tag):
            if op not in s.outputs:
                s[op].compute_inline()
            for tensor in op.input_tensors:
                if tensor.op.input_tensors:
                    traverse_inline(s, tensor.op, callback)
        callback(op)

    _traverse(final_op)
```

Reply:
It should be 
```
 for tensor in op.input_tensors:
                if tensor.op.input_tensors:
                    _traverse(tensor.op)
```
no?

Reply:
Aha, yes..